### PR TITLE
My Jetpack: Update product icon opacity in Product cards

### LINKED ISSUE #23511: My Jetpack: Make Product Card icons 40% opaque
To reduce some of its visual weight while still keeping the personality of each product intact.

### PR DESCRIPTION
**Pull request by oskosk**
Updates opacity of product icons to 40% on the product card section.

Fixes #23511

#### Changes proposed in this Pull Request:

* Updates IconWrapper to accept a property opacity defaulting to 1.
* Wrapps every icon on a G element using the opacity argument
image

#### Before
image

#### Jetpack product discussion
update/my-jetpack-product-card-button-behaviour

#### Does this pull request change what data or activity we track or use?
no
#### Testing instructions:
1. Checkout this branch, on a connected site
2. Visit My Jetpack
3. Confirm the icons look a 40% opaque.

### CODE REVIEW

**Review by kraftbj — DISMISSED** [review]

**Comment by oskosk** [comment]
Had to rebase after merging #23526

File: projects/packages/my-jetpack/_inc/components/connected-product-card/index.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const ConnectedProductCard = ( { admin, slug, showDeactivate } ) => {
 			name={ name }
 			description={ description }
 			status={ status }
-			icon={ <Icon /> }
+			icon={ <Icon opacity={ 0.4 } /> }
```
Comment (retrofox) [inline projects/packages/my-jetpack/_inc/components/connected-product-card/index.jsx:36]: why are we handling the opacity by attr instead of via CSS?

Comment (oskosk) [reply projects/packages/my-jetpack/_inc/components/connected-product-card/index.jsx:36]: For the same reason we're handling the color via attribute

**Review by retrofox — APPROVED** [review]
LGTM

**Comment by oskosk** [comment]
Had to rebase again after https://github.com/Automattic/jetpack/issues/23517 got merged

**Review by retrofox — APPROVED** [review]

**Pull request merged**